snyk_sync/utils.py

```diff
@@ -298,18 +302,26 @@ def ensure_dir(directory: Path) -> bool:
 
 def load_watchlist(cache_dir: Path) -> SnykWatchList:
     tmp_watchlist = SnykWatchList()
+    cache_data_errors = []
 
     if path.exists(f"{cache_dir}/data.json"):
```

Comment:
Do you need this as later on you do a jopen and if the file doesnt exist you should have caught it in the exception handling?